kali-persistence/01_kali-persistence.md

```diff
@@ -0,0 +1,14 @@
+# Create Kali linux 2018 live USB with persistence 
+
+	- First [download the latest](https://www.kali.org/downloads/) ISO kali linux version, for this project was used `kali-linux-2018.4-amd64.iso` image.

```

Comment:
```suggestion
First [download the latest](https://www.kali.org/downloads/) ISO kali linux version, for this project was used `kali-linux-2018.4-amd64.iso` image.
```

Reply:
The indentation made it looks as code block: [link](https://github.com/silent-dragonfly/docs/blob/7362ca1fcac3b8cdcb53b8ab4dc68fa25a5242d8/kali-persistence/01_kali-persistence.md)